qa/tasks/radosgw_admin.py

```diff
@@ -1106,6 +1106,39 @@ def task(ctx, config):
     # TESTCASE 'zonegroup-info', 'zonegroup', 'get', 'get zonegroup info', 'succeeds'
     (err, out) = rgwadmin(ctx, client, ['zonegroup', 'get'], check_status=True)
 
+    # TESTCASE 'add-storage-class', 'zonegroup', 'add', 'add storage class', 'succeeds'
+    storage_class = 'GLACIER'
+    (err, out) = rgwadmin(ctx, client, ['zonegroup', 'placement', 'add', '--rgw-zonegroup', 'default', '--placement-id', 'default-placement', '--storage-class', storage_class], check_status=True)
+ 
+
+    # TESTCASE 'add-storage-class', 'zone', 'add', 'add storage class', 'succeeds'
+    data_pool = '.rgw.buckets.2'
+    (err, out) = rgwadmin(ctx, client, ['zone', 'placement', 'add', '--rgw-zone', 'default', '--placement-id', 'default-placement', '--storage-class', storage_class, '--data-pool', data_pool], check_status=True)
+
+
+    # TESTCASE 'create bucket', 'put object with storage class', 'succeeds'
+    bucket = connection.create_bucket(bucket_name)
```

Comment:
```
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_teuthology_d9fdb2209e15b39d9f061fd85399f352ce0f0894/teuthology/run_tasks.py", line 105, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_teuthology_d9fdb2209e15b39d9f061fd85399f352ce0f0894/teuthology/run_tasks.py", line 83, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_0346e7b19ad2dc9dcaa46f28357e88a57ac89060/qa/tasks/radosgw_admin.py", line 1121, in task
    bucket = connection.create_bucket(bucket_name)
AttributeError: 'NoneType' object has no attribute 'create_bucket'
```

Reply:
ex. http://qa-proxy.ceph.com/teuthology/cbodley-2024-02-09_19:28:06-rgw-wip-cbodley2-testing-distro-default-smithi/7554194/teuthology.log from https://pulpito.ceph.com/cbodley-2024-02-09_19:28:06-rgw-wip-cbodley2-testing-distro-default-smithi/

Reply:
i now see it fail here with:
```
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/teuthology/run_tasks.py", line 105, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/teuthology/run_tasks.py", line 83, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_02db01d0a75058abf10a307d548d1e1e71b47eeb/qa/tasks/radosgw_admin.py", line 1110, in task
    bucket = connection.create_bucket(bucket_name)
  File "/home/teuthworker/src/git.ceph.com_teuthology_b1dac5519c57c269ea98a22fb7729016a1d0e4d2/virtualenv/lib/python3.8/site-packages/boto/s3/connection.py", line 627, in create_bucket
    raise self.provider.storage_response_error(
boto.exception.S3ResponseError: S3ResponseError: 403 Forbidden
<?xml version="1.0" encoding="UTF-8"?><Error><Code>InvalidAccessKeyId</Code><Message></Message><RequestId>tx00000720387af4375c46c-0065d22ecd-4224-default</RequestId><HostId>4224-default-default</HostId></Error>
```
ex. http://qa-proxy.ceph.com/teuthology/cbodley-2024-02-16_18:38:34-rgw-wip-64173-distro-default-smithi/7562752/teuthology.log from https://pulpito.ceph.com/cbodley-2024-02-16_18:38:34-rgw-wip-64173-distro-default-smithi/